ddtrace/_monkey.py

```diff
@@ -163,8 +163,18 @@ def on_import(hook):
                 "tracers", "integration_errors", 1, (("integration_name", module), ("error_type", type(e).__name__))
             )
         else:
+            version = ""
             imported_module.patch()
-            telemetry_writer.add_integration(module, True, PATCH_MODULES.get(module) is True, "")
+            if hasattr(imported_module, "get_version"):
+                version = imported_module.get_version()
+                if type(version) == dict:
```

Comment:
`get_version()` should return a consistent type (either string or dict whatever is the most common). Maybe should should a second optional method `get_versions()` that returns a dict. 

Or we can replace `get_version` with `get_versions` for one consistent api.